ble/Gap.h

```diff
@@ -911,6 +926,10 @@ class Gap {
     template<typename T>
     void onDisconnection(T *tptr, void (T::*mptr)(const DisconnectionCallbackParams_t*)) {disconnectionCallChain.add(tptr, mptr);}
 
+    DisconnectionEventCallbackChain_t& onDisconnection() {
```

Comment:
please add a documentation header. it might be good to add a code-example showing its use